donut/modules/newsgroups/templates/post.html

```diff
@@ -1,5 +1,17 @@
 {% extends "layout.html" %}
 {% block page %}
+  <script src='https://cdn.tiny.cloud/1/n8e5bron77mbkmlvy2i3tojaajwdsawb0b5kvet27prot9fq/tinymce/5/tinymce.min.js' referrerpolicy="origin">
```

Comment:
Are there other downsides besides additional dependencies? Because tinymce supports colors and nicer UI than editor, and can provide styled messages on the newsgroup page as long as we insert the html message in the db.

Reply:
Okay, that's fair. Forgot about styling

Reply:
Don't think we want to put HTML in the DB because of XSS concerns